cmd/tailscale/cli/configure-kube.go

```diff
@@ -44,6 +44,16 @@ See: https://tailscale.com/s/k8s-auth-proxy
 
 // kubeconfigPath returns the path to the kubeconfig file for the current user.
 func kubeconfigPath() string {
+	if kubeconfig := os.Getenv("KUBECONFIG"); kubeconfig != "" {
+		// Linux and Mac: ":"-delimited, Windows: ";"-delimited
+		// just handle both
+		if strings.ContainsAny(kubeconfig, ":;") {
+			kubeconfig, _, _ = strings.Cut(kubeconfig, ":")
+			kubeconfig, _, _ = strings.Cut(kubeconfig, ";")
+		}
+		return kubeconfig
```

Comment:
use https://pkg.go.dev/path/filepath#SplitList instead? That's its job?

Reply:
Done.
I had originally avoided it, but I don't remember the reasoning anymore, so this is perfectly reasonable.